### ISSUE TITLE
haddock error: Puppet/Parser/Types.hs:241:5: error: parse error on input ‘deriving’

### ISSUE THREAD

**Issue by juhp** [bartavelle/language-puppet#208]
```
:
Running Haddock for language-puppet-1.3.8...
Preprocessing library language-puppet-1.3.8...
Warning: The documentation for the following packages are not installed. No
links will be generated to these packages: array-0.5.1.1, base-4.9.1.0,
binary-0.8.3.0, bytestring-0.10.8.1, containers-0.5.7.1, deepseq-1.4.2.0,
directory-1.3.0.0, fail-4.9.0.0, filepath-1.4.1.1, ghc-boot-th-8.0.2,
ghc-prim-0.5.0.0, integer-gmp-1.0.0.1, pretty-1.1.3.3, process-1.4.3.0,
template-haskell-2.11.1.0, time-1.6.0.1, transformers-0.5.2.0, unix-2.7.2.1

Puppet/Parser/Types.hs:241:5: error:
    parse error on input ‘deriving’
Haddock coverage:
   0% (  0 /  5) in 'SafeProcess'
:
```

**Comment by bartavelle** [comment]
Well spotted, I will fix that.

**Comment by bartavelle** [comment]
Published v1.3.8.1

**Comment by juhp** [comment]
Thanks!
